src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -303,10 +318,18 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
         return setShouldShowStreamingAcknowledgement(true)
       if (isArbitrumBridgeTradeQuoteOrRate(activeQuote) && activeQuote.direction === 'withdrawal')
         return setShouldShowArbitrumBridgeAcknowledgement(true)
-
+      if (isThorchainSwapperEnabled && activeQuote?.steps[0]?.source === SwapperName.Thorchain) {
```

Comment:
blocking: `THORCHAIN_LONGTAIL_SWAP_SOURCE` and `THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE` too, though fine to use Array.prototype.includes() on `SwapperName.Thorchain` for the sake of simplicity

Reply:
Done in [2446009](https://github.com/shapeshift/web/pull/8718/commits/2446009104efd392993d9893fa0161f4e9ed33c1)